dex/src/main/scala/com/wavesplatform/dex/grpc/integration/clients/MatcherExtensionAssetsWatchingClient.scala

```diff
@@ -40,6 +41,8 @@ class MatcherExtensionAssetsWatchingClient(
       Task.fromFuture(saveAssetsDescription(assets)).map(_ => update)
     }
 
+  def get: WavesBlockchainClient = underlying
```

Comment:
It'd better to add:
```scala
def status: Status
```
to `WavesBlockchainClient`. 

You implemented it in `CombinedWavesBlockchainClient` already.